nw/dialogs/about.py

```diff
@@ -181,6 +181,7 @@ def _fillAboutPage(self):
                 ("Français", "Jan Lüdke (jyhelle)"),
                 ("Norsk Bokmål", "Veronica Berglyd Olsen"),
                 ("Português", "Bruno Meneguello"),
+                ("Simplified Chinese", "Qianzhi Long"),
```

Comment:
The entry "Simplified Chinese" should not be in English, but in the local name and language like the other languages.

Reply:
Hi, @vkbo

Which way is convenient to you?
- pull vkbo/dev, and change the branch name, and fix
- Or just fix directly in my own dev 

Let me know:)

Reply:
Just fix directly in your own dev is fine. I managed to work around the name conflict on my end, so the problem is solved for this PR. But please make a new branch next time :)